governance/polls/DAO Masons Poll 2 Exercise.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Derecognition of Aligned Delegate Clive
+summary: Signal your support or opposition to derecognizing Aligned Delegate Clive due to evidence of compromised identity.
+discussion_link: https://github.com/makerdao/community/blob/master/governance-exercises/poll-exercise-02.md
+parameters:
+  input_format:
+    type: single-choice
+    options: [1, 2]
+    abstain: [0]
+  victory_conditions:
+    - {
+        type: "and",
+        conditions: [{ type: approval }, { type: majority, percent: 50 }],
```

Comment:
See comment on other file. Pretty sure binary/plurality is equivalent here, and less complex.